### DISCUSSION TITLE
Mix of KFUS / 9FUS / IFUS on 12.x and 11.x Firmware - What are my options?

### DISCUSSION THREAD

**Discussion by jaimefordham** [roleoroleo/yi-hack-Allwinner-v2#662] [Q&A]
Hi everyone!

I'm tired of the new ad supported and constant "person detection has been enabled" marketing techniques as a way to get you to purchase a cloud subscription. 

I have a mix of Yi Cameras on different firmware versions:

- KFUS on 12.1.05.1
- 9FUS on 12.2.0.2
- IFUS on 11.1.26.1

I've seen that the hardware models are compatible but the firmware versions listed are below what my cameras are running.

What are my options for downgrading the firmware and applying the hack?

**Comment by roleoroleo** [comment]
You have to identify the model suffix of the cam (file prefix if you read the fw files).

- KFUS on 12.1.05 - model unknown at the moment, please send me a backup copy
- 9FUS on 12.2.0 - model unknown at the moment, please send me a backup copy
- IFUS on 11.1.26 - h30ga, if you want you can downgrade to 9.0.26

**Comment by jaimefordham** [comment]
Thanks!

How would I read the firmware files?

**Comment by roleoroleo** [comment]
https://github.com/roleoroleo/yi-hack-Allwinner-v2/wiki/Dump-your-backup-firmware-(SD-card)

**Comment by jaimefordham** [comment]
Thanks, done!

LINKS REMOVED

**Comment by roleoroleo** [comment]
KFUS on 12.1.05 - r30gb, if you want you can downgrade to 9.0.05
9FUS on 12.2.0 - y20ga (Allwinner), if you want you can downgrade to 8.2.0
IFUS on 11.1.26 - h30ga, if you want you can downgrade to 9.0.26

Pay attention: r30gb with 12.1.05 is not supported at the moment.
I will add the support asap.

**Comment by jaimefordham** [comment]
Thanks @roleoroleo you've been really helpful!

I guess the only question now is, where do I get the downgraded versions for KFUS, 9FUS and IFUS in order to downgrade?

**Comment by jaimefordham** [comment]
Does anyone know where I can get the downgraded versions for these platforms?

**Comment by roleoroleo** [comment]
About y20ga, I can help you.
